fdbserver/DataDistribution.actor.cpp

```diff
@@ -1829,22 +1793,49 @@ ACTOR Future<Void> auditStorageCore(Reference<DataDistributor> self,
 	lockInfo.prevWrite = self->lock.prevWrite;
 
 	try {
+		if (audit->getDDAuditContext() == DDAuditContext::Resume) {
```

Comment:
Do we have some checks in the following audit process of the persistent status of the audit?

Reply:
This part is outdated. I removed them